reproman/resource/docker_container.py

```diff
@@ -225,14 +225,7 @@ class DockerSession(POSIXSession):
     @borrowdoc(Session)
     def _execute_command(self, command, env=None, cwd=None):
         #command_env = self.get_updated_env(env)
-        if env:
-            raise NotImplementedError("passing env variables to docker session execution")
-
-        if cwd:
-            # TODO: implement
-            # raise NotImplementedError("handle cwd for docker")
-            lgr.warning("cwd is not handled in docker yet")
-            pass
+        command = self._prefix_command(command, env=env, cwd=cwd)
         # if command_env:
             # TODO: might not work - not tested it
             # command = ['export %s=%s' % k for k in command_env.items()] + command
```

Comment:
Since you've added support, can we drop this comment about a potential way to support it?  And I think get_updated_env is handled elsewhere, so we can drop the commented `command_env` assignment above.